lib/Helper/File.ts

```diff
@@ -49,6 +49,10 @@ export function exists(globPattern: string): boolean {
   }, true);
 }
 
+export function directoryExists(dirPath: string): boolean {
```

Comment:
Can you move this to `src/nextjs/utils.ts` instead? We're trying to phase out the `lib` directory as by now it contains only code from some older wizard flows which haven't been updated in a while.

Reply:
I will definitely do this, but I am currently traveling and I don’t have access to my computer for the coming weeks. I will do it as soon as I am back! 🤗